src/AppBundle/Controller/Admin/VoteSettingsController.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use AppBundle\Entity\VoteSettings;
+use AppBundle\Form\VoteSettingsType;
+
+/**
+ * VoteSettings controller.
+ *
+ * @Route("/admin/vote_settings")
+ */
+class VoteSettingsController extends Controller
+{
+    /**
+     * Lists all VoteSettings entities.
+     *
+     * @Route("/", name="vote_settings")
+     * @Method("GET")
+     * @Template()
+     */
+    public function indexAction()
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entities = $em->getRepository('AppBundle:VoteSettings')->findAll();
+
+        return array(
+            'entities' => $entities,
+        );
+    }
+    /**
+     * Creates a new VoteSettings entity.
+     *
+     * @Route("/", name="vote_settings_create")
+     * @Method("POST")
+     * @Template("AppBundle:VoteSettings:new.html.twig")
+     */
+    public function createAction(Request $request)
+    {
+        $entity = new VoteSettings();
+        $form = $this->createCreateForm($entity);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($entity);
+            $em->flush();
+
+            return $this->redirect($this->generateUrl('vote_settings_show', array('id' => $entity->getId())));
+        }
+
+        return array(
+            'entity' => $entity,
+            'form'   => $form->createView(),
+        );
+    }
+
+    /**
+     * Creates a form to create a VoteSettings entity.
+     *
+     * @param VoteSettings $entity The entity
+     *
+     * @return \Symfony\Component\Form\Form The form
+     */
+    private function createCreateForm(VoteSettings $entity)
+    {
+        $form = $this->createForm(new VoteSettingsType(), $entity, array(
+            'action' => $this->generateUrl('vote_settings_create'),
+            'method' => 'POST',
+        ));
+
+        $form->add('submit', 'submit', array('label' => 'Create'));
+
+        return $form;
+    }
+
+    /**
+     * Displays a form to create a new VoteSettings entity.
+     *
+     * @Route("/new", name="vote_settings_new")
+     * @Method("GET")
+     * @Template()
+     */
+    public function newAction()
```

Comment:
I think this is more uniq. In newAction I can something more, not only call createAction, more logic - this is simple